### PR TITLE
Issue 210

### PR DESCRIPTION
**Pull request by MoLi7**

### CODE REVIEW

File: R/DisaggregateFunctions.R [hunk 1]
```diff
@@ -491,9 +494,17 @@ disaggregateUseTable <- function (model, disagg, domestic = FALSE) {
...
   if(domestic) {
-    originalFD <-model$DomesticFinalDemand
+    if (model$specs$CommodityorIndustryType=="Commodity") {
+      originalFD <-model$DomesticFinalDemand
+    } else {
+      originalFD <- model$DomesticFinalDemandbyCommodity
+    }
   } else {
-    originalFD <-model$FinalDemand
+    if (model$specs$CommodityorIndustryType=="Industry") {
```
Comment (bl-young) [inline R/DisaggregateFunctions.R:503]: I think this is backwards? == Commodity?

Comment (MoLi7) [reply R/DisaggregateFunctions.R]: Yes, just corrected it [d53a963](https://github.com/USEPA/useeior/pull/211/commits/d53a963bd00d9524eccb1afadb89fc7a8330c82d) 😉

**Review by jvendries — COMMENTED** [review]
Changes allow model build of USEEIOv2.0-i-411 (industry) model and do not seem to conflict with other model builds.

**Comment by MoLi7** [comment]
Both the R-CMD and validation checks, including the one on industry 411 model, have passed, thanks to your contribution @jvendries , great work!

**Pull request merged**